build.gradle

```diff
@@ -26,7 +26,7 @@ apply plugin: net.fabricmc.loom.LoomGradlePlugin
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
-archivesBaseName = "cotton-resources"
+archivesBaseName = "cotton"
```

Comment:
? no?

Reply:
I agree with Falk. Even though we're eventually going to have the main repo and resources be one jar, this is just a terrible idea for organization before that point.